### ISSUE TITLE
ATM5 - v1.26 - Silent Gear Crimson Steel Hammer and Pick do not break blocks.

### ISSUE THREAD

**Issue by eliminat** [SilentChaos512/Silent-Gear#130]
## Versions

- **Silent Gear**: 1.7.0+93
- **Silent's Gems**: N/A
- **Silent Lib**: 4.6.1+49
- **Forge**: 31.2.0
- **Modpack**: ATM5 1.26 - Also reported there as I haven't been able to verify yet if the issue only occurs in this pack.
- **Optifine Installed**: No 

Video showing the issue: https://youtu.be/PWTBS0u0Xv0

Made a Silent Gear Crimson Steel hammer and pickaxe in 1.26
They don't seem to actually break blocks.
I'm unaware of any configuration that could cause this, and it's not spawn protection. Tested 11,000 blocks apart to verify not a location bug.
Also checked and have this issue with the Crimson Steel shovel and my Refined Iron Excavator that worked prior in ATM5 1.24 and 1.25.

Did not see anything in any of the logs, but let me know if there would be a useful one to provide.

Quick Edit: this is a private server of myself and a single friend. Not a single-player instance. Have not yet verified in a single-player instance. WIll try when I have another free moment.

**Comment by SilentChaos512** [comment]
Might be related to #126. Seems like something odd is happening with stats, but I haven't been able to reproduce this issue yet. Might be worth trying the `/sgear_recalculate @p` command if this doesn't happen consistently.

I would also like to see an `/sl_nbt item` export of the affected items if you could. Just make sure the issue is still happening before you run the command.

**Comment by eliminat** [comment]
Still happening.

here's the output. Haven't tried recalculate yet. Another person on the ATM5 github stated it's also happening to everyone on their server. 


[nbt_export_2020-05-29_15-04-09.zip](https://github.com/SilentChaos512/Silent-Gear/files/4704045/nbt_export_2020-05-29_15-04-09.zip)

**Comment by SilentChaos512** [comment]
Is it just items that were crafted before 1.7.0 that are having this issue? Does this happen if you craft an entirely new item with the same parts?

**Comment by eliminat** [comment]
Got materials for new Crimson Steel tools..

A new hammer doesn't seem to have the issue.
I also noticed the first hammer somehow has Flame Ward I on it, while the new one does not.
If I try to break blocks first, and they fail to break, and then switch to the new hammer the blocks I tried to break with the first hammer immediately break.

Tested with new Pick and Excavator, also seem good.
The strange thing is these were new tools when I first had the issue, hadn't been able to use them since making them. But so far no issues with the 2nd attempt (thought its only been a few minutes).
